src/james.crypto.md5.clss.pas

```diff
@@ -52,13 +57,37 @@ implementation
 { TMD5Stream }
 
 function TMD5Stream.GetStream: IDataStream;
+{$IFNDEF FPC}
+  function getMd5HashString(Value: string): string;
```

Comment:
Just found out Delphi XE8 and up have a native unit for several types of hashing, including MD5.
I changed the code to use it.
The advantage is, Indy is no longer required.
The disadvantage is current lack of support for older versions of Delphi.